src/entry.rs

```diff
@@ -248,8 +248,8 @@ mod tests {
 
         // First, verify entries
         let keypair = Keypair::new();
-        let tx0 = Transaction::new_timestamp(&keypair, Utc::now(), zero);
-        let tx1 = Transaction::new_signature(&keypair, Default::default(), zero);
+        let tx0 = Transaction::new_timestamp(&keypair, keypair.pubkey(), Utc::now(), zero);
```

Comment:
@garious in the bank.rs tests it actually uses it as intended,
```
 let tx = Transaction::new_timestamp(&mint.keypair(), pubkey, dt, bank.last_id());
```
because the contract context is under `pubkey`